libs/react-helpers/src/lib/local-logger.ts

```diff
@@ -0,0 +1,119 @@
+import * as Sentry from '@sentry/browser';
+import type { SeverityLevel, Scope } from '@sentry/browser';
+import type { Severity, Breadcrumb, Primitive } from '@sentry/types';
+
+type ConsoleArg = string | number | boolean | bigint | symbol | object;
+type ConsoleArgs = ConsoleArg[];
+
+interface LoggerConf {
+  application?: string;
+  tags?: string[];
+}
+
+const isPrimitive = (arg: ConsoleArg | undefined | null): arg is Primitive => {
+  return ['string', 'number', 'boolean', 'bigint', 'symbol'].includes(
+    typeof arg
+  );
+};
+
+export class LocalLogger {
+  tags: string[] = [];
+  application = 'trading';
+  constructor(conf: LoggerConf) {
```

Comment:
suggestion (non blocking): not sure if you looked at all at https://www.npmjs.com/package/loglevel

But it provides a nice way to set the log level with something like `logger.setLevel('info')`

I think it would be useful addition (with or without the library) to be able to call something like window.logging.setLevel('info') via the console so we can inspect deployed apps easily. By default this would be on 'silent' so no logs are shown.

Reply:
So I implemented something like such functionality. Please check if this is it